### PR TITLE
Installer Confirmation Screen

### PR DESCRIPTION
**Pull request by devishke-orange**
PR contains:
- Installer Confirmation Screen

### CODE REVIEW

File: installer/client/src/pages/ConfirmationScreen.vue [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  },
+  data() {
+    return {
+      isLoading: false,
+      database: {...databaseModel},
+      adminUserName: '',
+    };
+  },
+  beforeMount() {
+    this.http.getAll().then((response) => {
```
Comment (Super-Chama) [inline installer/client/src/pages/ConfirmationScreen.vue:102]: chain these and enable loading.

```
this.loading = true;
this.http.getAll().then((response) => {
   const {data} = response.data;
   this.database = {...databaseModel, ...data};
   return this.http.request({
      method: 'GET',
      url: '/installer/api/admin-user',
   })
}).then((response) => {
   const {data} = response.data;
   this.adminUserName = data.username;
   this.loading = false;
 });
```

File: installer/client/src/pages/ConfirmationScreen.vue [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        url: '/installer/api/admin-user',
+      })
+      .then((response) => {
+        const {data} = response.data;
+        this.adminUserName = data.username;
+      });
+  },
+  methods: {
+    onSubmit() {
+      // TODO navigate to install page
```
Comment (Super-Chama) [inline installer/client/src/pages/ConfirmationScreen.vue:118]: `navigate('installer/process');`

File: installer/client/src/pages/ConfirmationScreen.vue [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+      Confirmation
+    </oxd-text>
+    <br />
+    <oxd-text tag="p" class="orangehrm-installer-page-content">
+      All the information required for OrangeHRM installation collected in the
+      earlier steps are given below. On confirmation the installer will create
+      the database, database users, configuration file, etc
+    </oxd-text>
+    <br />
+    <oxd-text tag="h6" class="confirmation-text-header">Details</oxd-text>
```
Comment (Super-Chama) [inline installer/client/src/pages/ConfirmationScreen.vue:30]: use `h5` the size will come from it

File: installer/client/src/pages/ConfirmationScreen.vue (hunk 2 above)
Comment (devishke-orange) [reply installer/client/src/pages/ConfirmationScreen.vue]: changed

File: installer/client/src/pages/ConfirmationScreen.vue (hunk 1 above)
Comment (devishke-orange) [reply installer/client/src/pages/ConfirmationScreen.vue]: done

File: installer/client/src/pages/ConfirmationScreen.vue (hunk 3 above)
Comment (devishke-orange) [reply installer/client/src/pages/ConfirmationScreen.vue]: changed. but it seems too big compared to spec

**Pull request merged**